bundles/GoogleMarketingBundle/public/js/report/analytics/elementexplorer.js

```diff
@@ -262,5 +262,5 @@ pimcore.report.analytics.elementexplorer = Class.create(pimcore.report.abstract,
 // add to report broker
 if(pimcore.bundle && pimcore.bundle.customreports) {
```

Comment:
This check does not seem to work. I do not have the custom reports bundle installed and get the following error in the console `Uncaught TypeError: Cannot read properties of undefined (reading 'addGroup')`